app/channel_category_test.go

```diff
@@ -62,8 +62,10 @@ func TestSidebarCategory(t *testing.T) {
 		err = th.App.UpdateSidebarCategoryOrder(user.Id, th.BasicTeam.Id, actualOrder)
 		require.Nil(t, err, "Should update order successfully")
 
-		actualOrder[2] = "asd"
-		err = th.App.UpdateSidebarCategoryOrder(user.Id, th.BasicTeam.Id, actualOrder)
+		newOrder := make([]string, len(actualOrder))
```

Comment:
Can we add a small comment explaining why this is done? Mainly to prevent people looking at this code wondering if it can be removed.